processing/src/main/java/io/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -110,6 +110,21 @@ public void inspectRuntimeShape(RuntimeShapeInspector inspector)
   }
 
   @Override
+  public boolean hasMultipleValues(final String dimension)
+  {
+    if (isVirtualColumn(dimension)) {
+      return virtualColumns.getVirtualColumn(dimension).capabilities(dimension).hasMultipleValues();
+    }
+
+    final Column column = index.getColumn(dimension);
+    if (column == null || !columnSupportsFiltering(column)) {
```

Comment:
Ugh, sorry, this was my mistake. I changed it to remove the references to columns supporting filtering.